### PR TITLE
Sc-75

### PR DESCRIPTION
**Pull request by mbunico**

### CODE REVIEW

File: components/layout/JobsCard.vue [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+          <p class="text-sm" align="left">Fixed-price - Intermediate - Est. Budget: $5/hr - Posted 1 week ago</p>
+        </b-card> -->
+      </div>
+
+    </div>
+  </div>
+</template>
+
+<style>
+.description {
```
Comment (janlofredy) [inline components/layout/JobsCard.vue:35]: Don't use negative margins

File: components/layout/JobsLike.vue [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    const currentTab = (tabNumber) => jobsStore.setCurrentTab(tabNumber);
+    
+    return { jobsStore, currentTab, };
+  },
+};
+</script>
+
+<style>
+  .active{
+  border-bottom: 2px solid black;
```
Comment (janlofredy) [inline components/layout/JobsLike.vue:60]: use tailwind 
@apply border-b-2 border-black

**Review by janlofredy — CHANGES_REQUESTED** [review]

File: components/layout/JobsCard.vue (hunk 1 above)
Comment (mbunico) [reply components/layout/JobsCard.vue:35]: fixed

File: components/layout/JobsLike.vue (hunk 2 above)
Comment (mbunico) [reply components/layout/JobsLike.vue:60]: fixed

**Review by janlofredy — APPROVED** [review]

**Pull request merged**